smriprep/interfaces/templateflow.py

```diff
@@ -11,9 +11,13 @@
 class _TemplateFlowSelectInputSpec(BaseInterfaceInputSpec):
     template = traits.Str('MNI152NLin2009cAsym', mandatory=True, desc='Template ID')
     atlas = InputMultiObject(traits.Str, desc='Specify an atlas')
+    cohort = InputMultiObject(traits.Int, desc='Specify an atlas')
```

Comment:
I needed to update this bit. I haven't actually tested this with cohorts though.